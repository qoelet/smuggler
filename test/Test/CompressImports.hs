module Test.CompressImports where

import System.FilePath (isValid)
import System.FilePath.Posix (isAbsolute)
import System.Environment (setEnv)
import System.Environment.Blank (getEnvDefault, getArgs)

foo :: FilePath -> Bool
foo x = isValid x && isAbsolute x

bar :: IO ()
bar = do
  xs <- getArgs
  if not (null xs)
    then do
      x <- getEnvDefault "foo" (head xs)
      setEnv "bar" x
    else pure ()
