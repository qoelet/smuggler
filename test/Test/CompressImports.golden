module Test.CompressImports where

import System.FilePath (isValid,isAbsolute)
import System.Environment (setEnv,getArgs)
import System.Environment.Blank (getEnvDefault)

foo :: FilePath -> Bool
foo x = isValid x && isAbsolute x

bar :: IO ()
bar = do
  xs <- getArgs
  if not (null xs)
    then do
      x <- getEnvDefault "foo" (head xs)
      setEnv "bar" x
    else pure ()
